cmp/grid/GridModel.js

```diff
@@ -269,15 +259,20 @@ export class GridModel {
         return this.selModel.singleRecord;
     }
 
-    @action
-    setAgApi(agApi) {
-        this.agApi = agApi;
-    }
+    get compact() { return this.agGridModel.compact}
```

Comment:
The problem with trampolining these is that they also need documented -- I still wonder if not easier/more powerful to just reveal to the applications the underlying model as a getter, and to take in the constructor these options as a bundle....

I guess the change as is is more backward compatible, but I doubt many apps are actually setting *any* of these dynamically other than compact